### PR TITLE
Bn254

### PR DESCRIPTION
**Pull request by sczembor**
Closes #x

## Proposed Changes

- add implementation for `bn254` opcode

The `bn254` opcode has been postponed to TEALv8 or TEALv9. The name in go-algorand is currently `bn256` but this will most likely changed before release. Also there will be added support for `bls12-381` curve due to security concerns as discussed with developers on TEAL discord

## TODO

- [x] update change log

### CODE REVIEW

File: packages/runtime/test/src/interpreter/opcode-list.ts [hunk 1]
```diff
@@ -7472,4 +7475,83 @@ describe("Teal Opcodes", function () {
...
+			const pointABytes = Buffer.from(pointA, "hex");
+			const pointBBytes = Buffer.from(pointB, "hex");
+			const op = new Bn254Add([], 1);
+			stack.push(pointABytes);
+			stack.push(pointBBytes);
+			op.execute(stack);
+			assert.deepEqual(Buffer.from(expectedResult, "hex"), stack.pop());
+		});
+
+		it("Should throw an error when the points not in G1 (64 bytes lenght", function () {
```
Comment (rmeena840) [inline packages/runtime/test/src/interpreter/opcode-list.ts:7497]:
```suggestion
		it("Should throw an error when the points not in G1 (64 bytes length", function () {
```

**Review by rmeena840 — COMMENTED** [review]
Looks good. Check comment and linter.

File: packages/runtime/test/src/interpreter/opcode-list.ts (hunk 1 above)
Comment (rmeena840) [reply packages/runtime/test/src/interpreter/opcode-list.ts:7500]: I don't see the suggestion being committed. Please check

**Review by rmeena840 — COMMENTED** [review]
I don't see the suggested change.

File: package.json [hunk 2]
```diff
@@ -50,5 +50,8 @@
 			"eslint --format stylish --fix",
 			"prettier --write"
 		]
+	},
+	"dependencies": {
```
Comment (vuvoth) [inline package.json:54]: I think this should be in `packages/runtime/package.json`

**Review by vuvoth — COMMENTED** [review]
LGTM! You can merge after another member approval PR. Thank for good work!

**Review by rmeena840 — APPROVED** [review]

**Pull request merged**